modules/andes-core/client/src/main/java/org/wso2/andes/client/AMQConnectionDelegate_8_0.java

```diff
@@ -116,6 +116,15 @@ public ProtocolVersion makeBrokerConnection(BrokerDetails brokerDetail) throws A
             settings.setTrustStoreCertType(sslConfig.getCertType());
         }
 
+        if(settings.getKeyStorePath() == null && settings.getKeyStorePassword() ==null && settings
+                .getTrustStorePath() == null && settings.getTrustStorePassword() == null){
+            if (_logger.isDebugEnabled())
+            {
+                _logger.debug("Loading keystore and truststore from system properties");
```

Comment:
I think this information helps a lot in debugging ssl issues.
Therefore shall we make the log level to WARN?